modules/nw-sriov-network-attachment.adoc

```diff
@@ -1,36 +1,34 @@
 // Module included in the following assemblies:
 //
-// * networking/multiple_networks/configuring-sr-iov.adoc
+// * networking/hardware_networks/configuring-sriov-net-attach.adoc
 
 // Because of an existing issue in go-yaml, the strings 'on' and 'off'
 // are interpreted as booleans, not strings. The SR-IOV admission controller
 // will reject 'spoofCheck' and 'trust' if the values are not strings.
 // So these values must be explicitly quoted in the YAML.
 // https://github.com/go-yaml/yaml/issues/214
 
-[id="configuring-sr-iov-networks_{context}"]
+[id="nw-sriov-network-attachment_{context}"]
```

Comment:
Is it necessary to change this anchor? Typically we try to avoid file name/anchor changes after the docs are published, to eliminate any need for redirects and avoid breaking bookmarks.